### ISSUE TITLE
Postinstall fails on Windows

### ISSUE THREAD

**Issue by rarila** [psyrendust/grunt-eol#5]
In version 1.1.2 [Postinstall](https://github.com/psyrendust/grunt-eol/blob/master/package.json#L28) fails on Windows and Cygwin. Something like (sorry, I only have a non english message here atm)

> "-z" can’t be processed syntactically here

**Comment by rarila** [comment]
After looking at it, moving the `installhook` code translated to javascript into a (then renamed to postinstall.js or something like that) like that

``` javascript
if (parentFolderName === 'node_modules') {
   // Having the following translated to node.fs, which shouldn’t be a big deal, I guess :-)
   // if [ -f node_modules/angular-precommit/index.js ] && [ -d .git/hooks ]; then cp node_modules/angular-precommit/index.js .git/hooks/commit-msg; fi
}
```

and just have

```
"postinstall": "node ./postinstall.js",
```

in package.json then would work platform independent then?!

**Comment by psyrendust** [comment]
Great tip @rarila. I'll see about roping this in this afternoon.

**Comment by psyrendust** [comment]
@rarila can you give it a shot again with the latest in the develop branch? I'll cut another patch release if that works.

**Comment by rarila** [comment]
Tested and works great again! :+1: 
Thanks!

**Comment by psyrendust** [comment]
Great, I'll cut the release now.

**Comment by psyrendust** [comment]
Landed: https://github.com/psyrendust/grunt-eol/releases/tag/v1.1.3
Published to npm as `v1.1.3` - https://www.npmjs.com/package/grunt-eol
